Rakefile

```diff
@@ -237,6 +238,14 @@ def install_npm
   end
 end
 
+def ensure_code_studio_package
+  # never download if we build our own
+  return if CDO.use_my_code_studio
+
+  packager = S3Packaging.new('code-studio', code_studio_dir, dashboard_dir('public/code-studio-package'))
+  raise "No valid package found" unless packager.update_from_s3
```

Comment:
Note: This means if you don't use_my_code_studio, and sync to a point before we started uploading to s3 or to a point where we don't have a package for some reason, this will fail. Not sure if there's a way we can be smarter about this.